client/src/components/SearchFilters.js

```diff
@@ -507,8 +507,25 @@ export const searchFilters = function() {
     }
   }
 
+  const locationTypeOptions = [
+    Location.LOCATION_TYPES.ADVISOR_LOCATION,
+    Location.LOCATION_TYPES.PRINCIPAL_LOCATION,
+    Location.LOCATION_TYPES.PINPOINT_LOCATION,
+    Location.LOCATION_TYPES.GEOGRAPHICAL_AREA,
+    Location.LOCATION_TYPES.VIRTUAL_LOCATION
```

Comment:
Shouldn't we allow `Location.LOCATION_TYPES.PHYSICAL_LOCATION` in between these two?

Reply:
Even pinpoint location should't be allowed in the search according to @dspCarlson so I didn't even think of adding the physical location and we have two different missions to think of, so some further clarification would be great.

Reply:
We've decided not to add this one for now.